renovate.json

```diff
@@ -2,33 +2,32 @@
   "timezone": "America/New_York",
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:base",
+    "group:all"
   ],
-  "dependencyDashboard": true,
-  "includeForks": true,
   "logFileLevel": "trace",
   "enabledManagers": ["gomod"],
   "postUpdateOptions": ["gomodTidy"],
```

Comment:
I would add
```
"constraints": {
    "go": "1.18"
  },
```
to keep bumps compatible with or 1.18 limit.

Reply:
Also I would add `"dependencyDashboard": true,` to get pinned issue collecting all the pending bumps in a single place.

Reply:
I can re-add the dashboard. Fwiw I removed it but it is still getting created so it doesn't seem to matter: https://github.com/dprince/openstack-operator/issues/15